### PR TITLE
Create a basic CI setup #310

### PR DESCRIPTION
**Pull request by SelmanAY**
hi,

### what this feature does: 
  - using appveyor server creates a clean installation of three sql server installations (2008r2, 2012sp1, 2014)
  - runs the scripts to these instances
    * if installation fails the build will fail
    * these installations does not contains ReportServer database (so sp_BlitzRS fails to install) so in install.ps1 file I have checked if ReportServer databases exits, if so i run the script file. in short sp_BlitzRS is nor actually been installed or tested. 
    * test script files (in test.sql i have add a test for _ characters in parameters.)
    * if the build was a tagged build appveyor puts script files in a zip file and push this artifact back to github in releases tab. 

### what i changed : 
I only added files
  - appveyor.yml
  - install.ps1
  - test.sql
and added two more badges to the readme.md (i don't understand why it shows too many changes)

### how to use it :
in order to use this feature you need to 
1. create an appveyor account from appveyor.com
2. create a token from github. (https://github.com/settings/tokens) (for permissions look at https://www.appveyor.com/docs/deployment/github search for text 'GitHub authentication token')
3. encrypt the token using https://ci.appveyor.com/tools/encrypt
4. put encrypted data to appveyor.yml file

### CODE REVIEW

**Comment by BrentOzar** [comment]
Hi! Thanks for the contribution, this looks really cool. Excited to get this into place.

Right now, the pull request has conflicts. Github shows that sp_BlitzRS and sp_BlitzIndex have changed - if that wasn't intentional, go ahead and discard those changes. Same thing with README.md - merge the current changes from the dev branch into your code first.

Thanks, and I'm looking forward to getting this in!

File: sp_BlitzRS.sql [hunk 1]
```diff
@@ -964,4 +964,3 @@ IF @WhoGetsWhat = 1
 	ORDER BY EmailAddress, p.SubscriptionID  
 
 END
-
```
Comment (SelmanAY) [inline sp_BlitzRS.sql]: not intentional for testing install.ps1

File: sp_BlitzRS.sql [hunk 2]
```diff
@@ -8,7 +8,7 @@ GO
 
 IF OBJECT_ID('dbo.sp_BlitzRS') IS NULL
   EXEC ('CREATE PROCEDURE dbo.sp_BlitzRS AS RETURN 0;')
-GO
```
Comment (SelmanAY) [inline sp_BlitzRS.sql]: not intentional for testing install.ps1

**Comment by BrentOzar** [comment]
Errr, are you closing it to delete it? I'm confused. Want me to delete the pull request? (I'm kinda new to Github, but I don't think we're supposed to close issues that haven't been tested by anybody yet.)

**Comment by SelmanAY** [comment]
I'm new to githubs web editor. I can not manage to revert my changes using web client. 
i backed up my changes localy deleted my branch. I will fork the dev branch and do the changes again. 
then create a new pull request in a couple of minutes or so.

**Pull request closed without merging**